packages/askui-nodejs/src/core/model/custom-element-json.ts

```diff
@@ -67,10 +81,14 @@ export interface CustomElementJson {
    * only the green ones captured by `customImage` are filtered for using 'RGB'.
    * Defaults to 'grayscale'.
    *
-   * **Important**: This increases the prediction time quite a bit. So only use
-   * it when absolutely necessary.
+   * **Important**: The `imageCompareFormat` impacts the prediction time as
+   * well as quality. Although this highly depends on our use case, the other
```

Comment:
I wonder if we can remove the addressing of the user as we do not do it in the rest of the docs.

```suggestion
   * well as quality. Although this highly depends on the use case, the other
```